### PR TITLE
Removed setfacl. Using SETGID bit instead

### PR DESCRIPTION
**Pull request by vidarl**
Removed setfacl for the following reasons:
- Doesn't work with nfs ( if you want to put vardir on nfs for instance )
- Not all union file systems ( if any at all ) supports facls, thus making it impossible to store final file permissions in docker images

cons:
- User on host has to manually add himself to group 10000 and set umask 0002 if he wants to change code etc

### CODE REVIEW

**Comment by vidarl** [comment]
@andrerom : Ready for review
@jheba : ping

**Comment by andrerom** [comment]
where is set_permissions.sh executed?

**Comment by vidarl** [comment]
> where is set_permissions.sh executed?

https://github.com/vidarl/docker-ezplatform/blob/create_custom_distro_image/README.md

File: php/set_permissions.sh [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+    fi
+}
+
+function set_permissions_dev
+{
+    if [ "$PARAM_DEV" == "true" ]; then
+        chmod g+w -R /var/www
+        find /var/www -type d -exec chmod 2775 {} ';'
+        chown ez:ez -R /var/www
+        chown root -R /var/www/web/var
```
Comment (andrerom) [inline php/set_permissions.sh:105]: is it root that should have access to this folder?

**Comment by andrerom** [comment]
+1

**Comment by vidarl** [comment]
Merged to 7.0
Commits are cherry-picked to 5.5 and 5.6 branches

**Pull request merged**